index.html

```diff
@@ -231,5 +231,22 @@ <h3>LET'S CONNECT</h3>
               </ul>
              <div class="divider"></div>
        </section>
+       <footer class="contact-form">
+        <form class="contact-info" action="https://formspree.io/f/mwkyenjw" method="POST">
+            <h2>Contact me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            <div class="input-container">
+                <input type="text" name="Name" class="contact-input" maxlength="30" required placeholder="Write your name">
+                <input type="email" name="_replyTo" class="contact-input" placeholder="Write your email" required>
+                <textarea name="message" class="contact-textarea"  cols="30" rows="10" required maxlength="500" placeholder="Write your message here"></textarea>
+                <div class="works-flex-item get-it">
+                 <div class="btn-get no-padding">
+                   <a href="#" class="btn-get-it-touch">Get It Touch</a>
+                  </div>
```

Comment:
- [ ] Great job adding all the needed inputs and their properties, but right now, whenever you click on the button, it doesn't submit, it simply takes you back to the beginning of the page. This is happening because your button doesn't have the `type="submit"` as required by [formspree](https://formspree.io/), you can take a look here:

![image](https://user-images.githubusercontent.com/66967274/144045504-5d49bf3f-a460-4ae0-b9b0-2339bf048be0.png)